### PR TITLE
[NL] Update Taiko Ranking Criteria and Rules

### PR DESCRIPTION
**Pull request by Joppe27**
I updated the Dutch osu!taiko ranking article to be on par with the English version. I also updated the Dutch rules article and corrected some translations to make it more clear.

### CODE REVIEW

**Comment by Joppe27** [comment]
I don't know how to fix no-paragraph-content-indent , can someone help me on this? Thanks!

**Comment by Joehuu** [comment]
We are currently setting this up correctly, so don't worry about that for now.

File: wiki/Ranking_Criteria/osu!taiko/nl.md [hunk 1]
```diff
@@ -1,61 +1,181 @@
----
-outdated: true
----
-
 # osu!taiko Ranking Criteria
 
-Over deze regels en richtlijnen is gediscussieerd in het [osu!taiko Discussion Thread](https://osu.ppy.sh/community/forums/posts/1275323). Nieuwe regels en richtlijnen kunnen gesuggereerd worden in het [discussion forum](https://osu.ppy.sh/community/forums/87) dat zal worden geupdated, als de discussie wordt afgerond
+De osu!taiko ranking criteria zijn regels en richtlijnen die van toepassing zijn op het mappen van osu!taiko-specifieke moeilijkheidsgraden. Om een osu!taiko specifieke moeilijkheidsgraad ranked te krijgen, is het verplicht dat de creatie voldoet aan de genoemde criteria. Hoewel **alle regels onder alle omstandigheden moeten worden nageleefd**, mogen richtlijnen onder uitzonderlijke omstandigheden worden genegeerd. Deze uitzonderlijke omstandigheden moeten worden gerechtvaardigd door een uitgebreide uitleg over waarom de richtlijn is genegeerd en over waarom het niet negeren van deze richtlijn de algemene kwaliteit van de beatmap zal verstoren.
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!taiko/nl.md:3]:
```suggestion
De **osu!taiko ranking criteria** zijn regels en richtlijnen die van toepassing zijn op het mappen van osu!taiko-specifieke moeilijkheidsgraden. Om een osu!taiko specifieke moeilijkheidsgraad ranked te krijgen, is het verplicht dat de creatie voldoet aan de genoemde criteria. Hoewel **alle regels onder alle omstandigheden moeten worden nageleefd**, mogen richtlijnen onder uitzonderlijke omstandigheden worden genegeerd. Deze uitzonderlijke omstandigheden moeten worden gerechtvaardigd door een uitgebreide uitleg over waarom de richtlijn is genegeerd en over waarom het niet negeren van deze richtlijn de algemene kwaliteit van de beatmap zal verstoren.
```

I think you are missing the bolding of `osu!taiko ranking criteria`

File: wiki/Ranking_Criteria/osu!taiko/nl.md [hunk 2]
```diff
@@ -1,61 +1,181 @@
...
+- **Geleidelijke *slider velocity* veranderingen:** Een techniek die een geleidelijke overgang van lagere naar hogere / van hogere naar lagere *slider velocity* secties mogelijk maakt. Om dit effect te bereiken wordt gebruik gemaakt van overgangsnoten met variërende *slider velocities*.
+- **Improvisatie:** Wanneer er meer noten in een beatmap zijn dan in de eigenlijke muziek.
+- **Taiko sjabloonachtergrond *(Taiko template background)*:** Achtergrondafbeeldingen die de authentieke Taiko no Tatsujin gameplay simuleren. Ze bevatten doorgaans een zwarte balk die de artiest en de titel van het liedje in het wit onder het speelveld toont.
+
+## Algemeen
+
+Algemene regels en richtlijnen zijn van toepassing op elke osu!taiko moeilijkheidsgraad. Ritme-gerelateerde regels en richtlijnen zijn van toepassing op beatmaps met ongeveer 180 BPM en met 4/4 maatsoort. Als jouw muziek veel sneller of trager is, zullen sommige variabelen anders zijn, zoals aangegeven in [Scaling BPM on the Ranking Criteria](/wiki/Ranking_Criteria/Scaling_BPM).
+
+### Regels
```
Comment (MegaApplePi) [inline wiki/Ranking_Criteria/osu!taiko/nl.md:43]: The headings here is done wrong, it should be 
```
### General

#### Rules
```

**Pull request merged**